STATUS: INLINE SUCCESSFUL
MODIFIED: de.strullerbaumann.visualee.ui.graph.boundary.GraphCreatorTest.testBuildJSONNode
MODIFIED FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/ui/graph/boundary/GraphCreatorTest.java
CLEANER: de.strullerbaumann.visualee.ui.graph.boundary.TestGroup100Case0.test255
CLEANER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/ui/graph/boundary/TestGroup100Case0.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
ORIGINAL CLEANER SIZE: 15
NEW CLEANER SIZE: 2
ELAPSED TIME: 109.93700003623962
==========================
@@ -46,0 +46,4 @@
+ {
+     de.strullerbaumann.visualee.filter.boundary.FilterContainer filterContainer0 = de.strullerbaumann.visualee.filter.boundary.FilterContainer.getInstance();
+     filterContainer0.clear();
+ }
